base/test.jl

```diff
@@ -912,6 +953,8 @@ end
 #-----------------------------------------------------------------------
 # Legacy approximate testing functions, yet to be included
 
+# BEGIN TODO: deprecated in 0.6, delete in 1.0
```

Comment:
it would be a major pain to deal with deprecations if they were all done this way. please don't encourage this, move the code to deprecated.jl with an eval into the right module